tests/testApCorrMap.py

```diff
@@ -87,10 +83,10 @@ def testAccessors(self):
         self.assertEqual(list(collections.OrderedDict(self.map).values()), list(self.map.values()))
         self.assertEqual(list(collections.OrderedDict(self.map).items()), list(self.map.items()))
         self.assertEqual(list(collections.OrderedDict(self.map).keys()), list(self.map))
-        self.assertTrue("b" in self.map)
-        self.assertTrue("d" not in self.map)
+        self.assertIn("b", self.map)
+        self.assertNotIn("d", self.map)
         self.map["d"] = lsst.afw.math.ChebyshevBoundedField(self.bbox, numpy.random.randn(2, 2))
-        self.assertTrue("d" in self.map)
+        self.assertIn("d", self.map)
         self.assertIsNone(self.map.get("e"))
         self.assertRaisesLsstCpp(lsst.pex.exceptions.NotFoundError, self.map.__getitem__, "e")
```

Comment:
Also, if it is rewritten as:
```python
with self.assertRaises(lsst.pex.exceptions.NotFoundError):
    self.map["e"]
```
I think it's much clearer what is happening (assuming you don't really need that `__getitem__` call to be explicit).